src/test/java/posters/pageobjects/components/TopNavigation.java

```diff
@@ -61,7 +61,8 @@ public CategoryPage clickSubCategoryByNames(String categoryName, String subCateg
         categoryMenu.find(By.linkText(categoryName)).hover();
         // Clicks the subcategory with position @{subCategoryPosition}
         // belonging to the category with position @{categoryPosition}
-        categoryMenu.find(By.linkText(subCategoryName)).click();
+        // Sometimes there can be a timeout at this point.
+        categoryMenu.find(By.linkText(subCategoryName)).waitUntil(exist, 5000).click();
```

Comment:
shouldn't the condition for this case be visible, since an invisible element would also lead to an error?

Reply:
done